### DISCUSSION TITLE
GitHub API rate limit

### DISCUSSION THREAD

**Discussion by J-minkuk** [community/community#24901] [Apps, API and Webhooks]
<p>Hi, community members</p>
<p>I’m MinGOOD [<a href="https://github.com/MinGOODdev" rel="nofollow noopener">https://github.com/MinGOODdev</a> ]</p>
<p>I’m currently developing application services using the GitHub API.</p>
<p>so that, There’s something I want to discuss.</p>
<p>How can minimize the number of GitHub API calls or efficiently use them within a limited number of times?</p>

**Comment by lee-dohm** [comment, marked as answer]
<p>The API developer documentation includes recommendations on how to do this <a href="https://developer.github.com/v3/#conditional-requests" rel="nofollow noopener">via caching and using Conditional Requests</a>. That should get you started.</p>
<p>Let us know if you have any questions!</p>
